agent/src/agent/pipeline/config/jython_scripts/events_destination.py

```diff
@@ -20,6 +20,14 @@ def main():
         sdc.output.write(record)
 
 
+def send_offset(offset):
+    try:
+        res = requests.post(sdc.userParams['AGENT_OFFSET_URL'], json={'offset': offset})
+        res.raise_for_status()
+    except (requests.ConnectionError, requests.HTTPError) as e:
+        sdc.error.write(str(e))
```

Comment:
If I'm not mistaken, `sdc.error` takes two arguments: record and some message.
If you want just log the error, the `sdc.log.error` should be used
[Here](https://docs.streamsets.com/datacollector/latest/help/datacollector/UserGuide/Processors/Jython.html?hl=sdc.error) is a docs